examples/cmake-c-example/GlobalRegistration.c

```diff
@@ -47,17 +51,40 @@ int main( int argc, char* argv[] )
             mrStringFree( errorString );
             goto out_inputs;
         }
-        inputXfs[i] = mrMeshOrPointsXfFromPointCloud( inputs[i], NULL );
-    }
 
-    MRICPProperties params = mrICPPropertiesNew();
+        inputXfs[i] = mrMeshOrPointsXfFromPointCloud( inputs[i], NULL ); // or mrMeshOrPointsXfFromMesh for meshes
+
+        MRBox3f bbox = mrPointCloudComputeBoundingBox( inputs[i], NULL );
+        if ( !mrBox3fValid( &maxBBox ) || mrBox3fVolume( &bbox ) > mrBox3fVolume( &maxBBox ) )
+            maxBBox = bbox;
+    }
 
     MRMultiwayICPSamplingParameters samplingParams = mrMultiwayIcpSamplingParametersNew();
+    samplingParams.samplingVoxelSize = mrBox3fDiagonal( &maxBBox ) * 0.03f;
     samplingParams.cb = onProgress;
 
     MRMultiwayICP* icp = mrMultiwayICPNew( *inputXfs, inputNum, &samplingParams );
+
+    MRICPProperties params = mrICPPropertiesNew();
     mrMultiwayICPSetParams( icp, &params );
+
+    // gather statistics
     mrMultiwayICPUpdateAllPointPairs( icp, NULL );
+    size_t numSamples = mrMultiWayICPGetNumSamples( icp );
+    size_t numActivePairs = mrMultiWayICPGetNumActivePairs( icp );
+    printf( "Samples: %zu", numSamples );
+    printf( "Active point pairs: %zu", numActivePairs );
+    if ( numActivePairs > 0 )
+    {
+        double p2ptMetric = mrMultiWayICPGetMeanSqDistToPoint( icp, NULL );
+        double p2ptInaccuracy = mrMultiWayICPGetMeanSqDistToPoint( icp, &p2ptMetric );
+        printf( "RMS point-to-point distance: %f ± %f", p2ptMetric, p2ptInaccuracy );
+
+        double p2plMetric = mrMultiWayICPGetMeanSqDistToPlane( icp, NULL );
+        double p2plInaccuracy = mrMultiWayICPGetMeanSqDistToPlane( icp, &p2plMetric );
+        printf( "RMS point-to-plane distance: %f ± %f", p2plMetric, p2plInaccuracy );
+    }
+
     MRVectorAffineXf3f* xfs = mrMultiwayICPCalculateTransformations( icp, NULL );
```

Comment:
also whould be nice to print update info after `mrMultiwayICPCalculateTransformations`